tests/TestCase/Validation/ValidatorTest.php

```diff
@@ -563,7 +563,7 @@ public function testErrorsWithLastRule()
         $errors = $validator->errors(['email' => 'not an email!']);
         $expected = [
             'email' => [
-                'alpha' => 'The provided value is invalid'
+                'alpha' => 'alpha'
```

Comment:
The default `The provided value is invalid'` is now gone. Instead you get the rule name now if you haven't explicitly specified `'message'` for that rule.